### ISSUE TITLE
Prettier extension is not longer working in vscode Insiders.

### ISSUE THREAD

**Issue by ymulenll** [Microsoft/vscode#68155]
Issue Type: <b>Bug</b>

After I updated vscode insiders a couple of days ago, `Prettier` extension stopped working, I also try the command to Format Document and nothing happens.

VS Code version: Code - Insiders 1.32.0-insider (45c7f29565821668f4d010b25e3289953bd9ac71, 2019-02-07T10:25:31.179Z)
OS version: Windows_NT x64 10.0.17134

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8550U CPU @ 1.80GHz (8 x 1992)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: enabled<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Memory (System)|15.84GB (1.58GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (23)</summary>

Extension|Author (truncated)|Version
---|---|---
npm-intellisense|chr|1.3.0
bracket-pair-colorizer-2|Coe|0.0.25
vscode-eslint|dba|1.8.0
es7-react-js-snippets|dsz|2.0.0
gitlens|eam|9.5.0
EditorConfig|Edi|0.12.8
vscode-npm-script|eg2|0.3.5
prettier-vscode|esb|1.8.1
auto-close-tag|for|0.5.6
auto-rename-tag|for|0.0.15
vscode-pull-request-github|Git|0.3.2
vscode-mocha-test-adapter|hbe|1.6.0
vscode-test-explorer|hbe|2.4.0
dotenv|mik|1.0.1
vscode-kubernetes-tools|ms-|0.1.16
debugger-for-chrome|msj|4.11.1
vscode-jest|Ort|2.9.2
vscode-docker|Pet|0.5.2
material-icon-theme|PKi|3.6.2
vscode-graphql|Pri|0.1.6
vscode-yaml|red|0.2.1
sass-indented|rob|1.5.1
vscode-multiclip|sle|0.1.5

(1 theme extensions excluded)

</details>

**Comment by octref** [comment]
- Open a JS file
- Run command `Developer: Print available formatters`
- Help -> Toggle Developer Tools

What's the output?

**Comment by gabygandrade** [comment]
I'm also seeing this issue, with VSCode version 1.31.0 

I only have 2 extensions installed:
```
Prettier: v1.81.1
TSLint: v1.0.43
```

I have the following in my `UserSettings.json` file (and nothing in my workspace settings):
```
{
  "editor.formatOnSave": true,
  "prettier.tslintIntegration": true
}
```

When I go into the Help -> Toggle Developer Tools I see this:

```
ERR prettierInstance.getSupportInfo is not a function: TypeError: prettierInstance.getSupportInfo is not a function
	at getSupportLanguages (/Users/gaby.andrade/.vscode/extensions/esbenp.prettier-vscode-1.8.1/out/src/utils.js:43:29)
	at Object.getParsersFromLanguageId (/Users/gaby.andrade/.vscode/extensions/esbenp.prettier-vscode-1.8.1/out/src/utils.js:11:22)
	at /Users/gaby.andrade/.vscode/extensions/esbenp.prettier-vscode-1.8.1/out/src/PrettierEditProvider.js:46:40
	at Generator.next (<anonymous>)
	at __awaiter (/Users/gaby.andrade/.vscode/extensions/esbenp.prettier-vscode-1.8.1/out/src/PrettierEditProvider.js:7:71)
	at new Promise (<anonymous>)
	at __awaiter (/Users/gaby.andrade/.vscode/extensions/esbenp.prettier-vscode-1.8.1/out/src/PrettierEditProvider.js:3:12)
	at format (/Users/gaby.andrade/.vscode/extensions/esbenp.prettier-vscode-1.8.1/out/src/PrettierEditProvider.js:40:12)
	at PrettierEditProvider._provideEdits (/Users/gaby.andrade/.vscode/extensions/esbenp.prettier-vscode-1.8.1/out/src/PrettierEditProvider.js:158:16)
	at PrettierEditProvider.provideDocumentFormattingEdits (/Users/gaby.andrade/.vscode/extensions/esbenp.prettier-vscode-1.8.1/out/src/PrettierEditProvider.js:152:21)
	at /private/var/folders/sm/86680nw13j56h7clmbqdl42m0000gp/T/AppTranslocation/7BDDDEFB-5F73-4DF4-AACA-6EE7201D7117/d/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:543:107
	at t.asPromise (/private/var/folders/sm/86680nw13j56h7clmbqdl42m0000gp/T/AppTranslocation/7BDDDEFB-5F73-4DF4-AACA-6EE7201D7117/d/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:50:385)
	at new Promise (<anonymous>)
	at Object.t.asPromise (/private/var/folders/sm/86680nw13j56h7clmbqdl42m0000gp/T/AppTranslocation/7BDDDEFB-5F73-4DF4-AACA-6EE7201D7117/d/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:50:353)
	at e.provideDocumentFormattingEdits (/private/var/folders/sm/86680nw13j56h7clmbqdl42m0000gp/T/AppTranslocation/7BDDDEFB-5F73-4DF4-AACA-6EE7201D7117/d/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:543:67)
	at /private/var/folders/sm/86680nw13j56h7clmbqdl42m0000gp/T/AppTranslocation/7BDDDEFB-5F73-4DF4-AACA-6EE7201D7117/d/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:558:542
	at e._withAdapter (/private/var/folders/sm/86680nw13j56h7clmbqdl42m0000gp/T/AppTranslocation/7BDDDEFB-5F73-4DF4-AACA-6EE7201D7117/d/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:553:292)
	at e.$provideDocumentFormattingEdits (/private/var/folders/sm/86680nw13j56h7clmbqdl42m0000gp/T/AppTranslocation/7BDDDEFB-5F73-4DF4-AACA-6EE7201D7117/d/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:558:504)
	at t._doInvokeHandler (/private/var/folders/sm/86680nw13j56h7clmbqdl42m0000gp/T/AppTranslocation/7BDDDEFB-5F73-4DF4-AACA-6EE7201D7117/d/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:644:118)
	at t._invokeHandler (/private/var/folders/sm/86680nw13j56h7clmbqdl42m0000gp/T/AppTranslocation/7BDDDEFB-5F73-4DF4-AACA-6EE7201D7117/d/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:643:744)
	at t._receiveRequest (/private/var/folders/sm/86680nw13j56h7clmbqdl42m0000gp/T/AppTranslocation/7BDDDEFB-5F73-4DF4-AACA-6EE7201D7117/d/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:642:256)
	at t._receiveOneMessage (/private/var/folders/sm/86680nw13j56h7clmbqdl42m0000gp/T/AppTranslocation/7BDDDEFB-5F73-4DF4-AACA-6EE7201D7117/d/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:641:155)
	at /private/var/folders/sm/86680nw13j56h7clmbqdl42m0000gp/T/AppTranslocation/7BDDDEFB-5F73-4DF4-AACA-6EE7201D7117/d/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:638:960
	at /private/var/folders/sm/86680nw13j56h7clmbqdl42m0000gp/T/AppTranslocation/7BDDDEFB-5F73-4DF4-AACA-6EE7201D7117/d/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:38:994
	at e.fire (/private/var/folders/sm/86680nw13j56h7clmbqdl42m0000gp/T/AppTranslocation/7BDDDEFB-5F73-4DF4-AACA-6EE7201D7117/d/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:44:113)
	at a (/private/var/folders/sm/86680nw13j56h7clmbqdl42m0000gp/T/AppTranslocation/7BDDDEFB-5F73-4DF4-AACA-6EE7201D7117/d/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:170:213)
	at Socket.f._socketDataListener (/private/var/folders/sm/86680nw13j56h7clmbqdl42m0000gp/T/AppTranslocation/7BDDDEFB-5F73-4DF4-AACA-6EE7201D7117/d/Visual Studio Code.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:170:434)
	at Socket.emit (events.js:182:13)
	at addChunk (_stream_readable.js:279:12)
	at readableAddChunk (_stream_readable.js:264:11)
	at Socket.Readable.push (_stream_readable.js:219:10)
	at Pipe.onread (net.js:636:20)
```

**Comment by poojadamidi** [comment]
I'm also seeing this issue, with VSCode version 1.31.0. I installed older versions and it's still a problem.

**Comment by ymulenll** [comment]
> * Open a JS file
> * Run command `Developer: Print available formatters`
> * Help -> Toggle Developer Tools
> 
> What's the output?

This is what I get:
**Range Formatters** 
    TypeScript and JavaScript Language Features
    Prettier - Code formatter
**Document Formatters**
    Prettier - Code formatter

**Comment by ymulenll** [comment]
I also uninstalled Prettier, and now I only have `TypeScript and JavaScript Language Features` as the formatter, and the Format Document or Format Selection are still not working.
But I see these errors on the developer tools/console:
![image](https://user-images.githubusercontent.com/3630913/52477986-9fe77600-2b82-11e9-81e9-060b528bd5c0.png)

**Comment by octref** [comment]
@gabygandrade @poojadamidi You should report that bug to https://github.com/prettier/prettier-vscode

@ymulenll Your problem is a bit peculiar. It seems your installation is corrupt. @mjbvz Can you take a look?

If this is the output:
```
**Range Formatters**
TypeScript and JavaScript Language Features
Prettier - Code formatter
**Document Formatters**
Prettier - Code formatter
```

Then "Format Document" should use prettier. Maybe prettier extension is failing to format.

**Comment by ymulenll** [comment]
@octref yes, I thought my problem was with Prettier, that's in the title of this issue. But I also uninstalled Prettier and my problem persists. Maybe it is because of the vscode-insiders update that corrupted my vscode typescript-language-features, because Format Document was working properly before.

**Comment by ymulenll** [comment]
> I also uninstalled Prettier, and now I only have `TypeScript and JavaScript Language Features` as the formatter, and the Format Document or Format Selection are still not working.
> But I see these errors on the developer tools/console:
> ![image](https://user-images.githubusercontent.com/3630913/52477986-9fe77600-2b82-11e9-81e9-060b528bd5c0.png)

I don't have the command folder inside, for that, I get all those console errors. `c:\Users\ymule\AppData\Local\Programs\Microsoft VS Code Insiders\resources\app\extensions\typescript-language-features\dist\`

**Comment by ymulenll** [comment]
@octref yes, I see an open issue in prettier-vscode repo that looks similar to this:
https://github.com/prettier/prettier-vscode/issues/704

I am still not sure about why I am getting errors because of not having this commands folder:
c:\Users\ymule\AppData\Local\Programs\Microsoft VS Code Insiders\resources\app\extensions\typescript-language-features\dist\commands

Thanks!

**Comment by mjbvz** [comment]
@ymulenll Try reinstalling VS Code. Missing internal scripts suggest that your extension is corrupt. On windows, misbehaving antivirus software is one common cause of this

**Comment by Mirakurun** [comment]
Similar issue here except for .vue files only.  Fixed problem by downgrading to 1.30.2

OS: Win7

**Comment by octref** [comment]
Please follow upstream issue: https://github.com/prettier/prettier-vscode/issues/704.
